app/controllers/forgot_passwords_controller.rb

```diff
@@ -0,0 +1,26 @@
+class ForgotPasswordsController < ApplicationController
+  
+  def new; end
+  
+  def create
+    user = User.where(email_address: params[:email_address]).first
+    
+    if user
+      user.generate_token
+      #binding.pry
```

Comment:
Sorry, just noticed I still had that in there when I was closing the windows after pushing the code and have already taken it out.